src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -2203,6 +2335,41 @@ type Plan {
     debugOutput: debugOutput!
 }
 
+type PlanConnection {
+    searchDateTime: PlanDateTime
+
+    locations: PlanPlaces
+
+    routingErrors: [RoutingError!]!
+
+    debugInformation: PlanDebugInformation!
+
+    edges: [PlanEdge]
+
+    pageInfo: PageInfo!
+}
+
+type PlanDateTime {
+    date: DateTime
```

Comment:
Should we use an OffsetDateTime?

Reply:
Yes.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/pull/5185/commits/910717a739f12e99b3a445c338fd24a8e4b73637